### DISCUSSION TITLE
add support for Torus as a custom provider

### DISCUSSION THREAD

**Discussion by QEDK** [wevm/wagmi#1016] [Ideas]
Currently, there is no support for social-based login wallet providers. Would be great to have Torus (Web3Auth) support similar to Blocknative Onboard SDK.

**Comment by YZhenY** [comment]
https://www.npmjs.com/package/@web3auth/web3auth-wagmi-connector theres one for web3auth - for torus wallet that will come soon too

**Comment by recidive** [comment]
It's here https://github.com/Web3Auth/web3auth-wagmi-connector !!

**Comment by tonyfung99** [comment]
do we have the direct torus provider without using web3auth?
